src/praisonai-agents/tests/unit/test_async_memory.py

```
@@ -157,28 +157,29 @@ def worker(thread_id: int):
+        import praisonaiagents
         
         errors = []
+        results = []
         
-        def access_cache():
+        def access_lazy_attr():
             try:
-                cache = _get_lazy_cache()
-                assert isinstance(cache, dict)
+                results.append(praisonaiagents.Task)
             except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)